### ISSUE TITLE
No module named "optlang"

### ISSUE THREAD

**Issue by ossdev07** [opencobra/cobrapy#912]
#### Problem description:

Module not found named "Optlang" which is already installed and all dependencies.

 **Explaination**:
I already unistall the package Optlang and again installed for upgrade version but always its shows requirements already there. In Conftest.py, After given the exact path of Optlang package but is shown exactly same issue  in aarch64 perhaps in x86 is going good after using tox -e py36 command.

#### Actual Output
```
.tox/py36/lib/python3.6/site-packages/_pytest/config/__init__.py:456: in _importconftest
    return self._conftestpath2mod[key]
E   KeyError: PosixPath('/work/cobrapy/cobra/test/conftest.py')

During handling of the above exception, another exception occurred:
.tox/py36/lib/python3.6/site-packages/_pytest/config/__init__.py:462: in _importconftest
    mod = conftestpath.pyimport()
.tox/py36/lib/python3.6/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
cobra/__init__.py:36: in <module>
    from cobra.core import (
cobra/core/__init__.py:5: in <module>
    from cobra.core.configuration import Configuration
cobra/core/configuration.py:15: in <module>
    from cobra.exceptions import SolverNotFound
cobra/exceptions.py:5: in <module>
    import optlang.interface
E   ModuleNotFoundError: No module named 'optlang'
.tox/py36/lib/python3.6/site-packages/_pytest/config/__init__.py:456: in _importconftest
    return self._conftestpath2mod[key]
E   KeyError: PosixPath('/work/cobrapy/cobra/test/conftest.py')

During handling of the above exception, another exception occurred:
.tox/py36/lib/python3.6/site-packages/_pytest/config/__init__.py:462: in _importconftest
    mod = conftestpath.pyimport()
.tox/py36/lib/python3.6/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
cobra/__init__.py:36: in <module>
    from cobra.core import (
cobra/core/__init__.py:5: in <module>
    from cobra.core.configuration import Configuration
cobra/core/configuration.py:15: in <module>
    from cobra.exceptions import SolverNotFound
cobra/exceptions.py:5: in <module>
    import optlang.interface
E   ModuleNotFoundError: No module named 'optlang'
```
#### Expected Output
```
You can refer the below link :
https://travis-ci.org/opencobra/cobrapy/jobs/587693480
```

**Comment by Midnighter** [comment]
Did you somehow uninstall the package `swiglpk`?

**Comment by ossdev07** [comment]
No , I didn't unistall the package swiglpk.

**Comment by Midnighter** [comment]
I don't understand your explanation very well. So you cloned the repository and ran the test suite through tox?

```
git clone https://github.com/opencobra/cobrapy.git
cd cobrapy
tox -e py36
```

If that's the case, can you please use `tox -e py36 --recreate`, please?

**Comment by cdiener** [comment]
This looks stale. Feel free to reopen if the problem persists,
